src/02-common-principles.md

```diff
@@ -56,6 +56,7 @@ misunderstanding we clarify them here.
     1.  `ieeg` (intracranial electroencephalography)
     1.  `beh` (behavioral)
     1.  `pet` (positron emission tomography)
+    1.  `microscopy`
```

Comment:
was the choice of going with full name (of length 10) discussed? IMHO it should be `mic` or `micr` or alike, so let me 

```suggestion
    1.  `micr`
```

some words in defense of such proposal:
- although descriptive, it would  contribute all 10 characters to every data file path:
   - more typing for those who can't complete ATM
   - would be inconsistent with the other data types being abbreviated 
- note that "perfusion" (length 9, which is less than 10) has chosen to go with shortened (`perf`) and `behavioral` (10) also was shortened

Reply:
As mentioned in https://github.com/bids-standard/bids-specification/pull/881#discussion_r754308168, we'll go with `micr` that seems to be the preferred choice.

Reply:
Great, thank you! I will resolve this conversation whenever some commit (my suggestion or not) changing that here is pushed